lib/takeaway.rb

```diff
@@ -1,24 +1,37 @@
 class Takeaway
 
+    attr_accessor :order, :menu
+
+MENU = {  
+        :Beer => 4.00,
+        :Cola => 2.50,
+        :Tap_water => 0.50,
+        :Burger => 6.00,
+        :Veggie_burger => 5.00,
+        :Chips => 2.00,
+        :Salad => 3.50 
+    }
+
     def initialize
         @order = {}
-        @menu = {  
-            :Beer => 4.00,
-            :Cola => 2.50,
-            :Tap_water => 0.50,
-            :Burger => 6.00,
-            :Veggie_burger => 5.00,
-            :Chips => 2.00,
-            :Salad => 3.50 
-        }
     end
 
     def view_menu
-        @menu
+        MENU
```

Comment:
Perhaps you could consider making a puts statement so it prints the menu to screen?